src/lib.rs

```diff
@@ -188,6 +189,75 @@ impl FixedBitSet
     {
         &mut self.data
     }
+
+    /// Iterates over all set bits, i.e. all '1's.
+    #[inline]
+    pub fn iter_ones<'a>(&'a self) -> IterOnes<'a> {
```

Comment:
Use lifetime elision, no need to spell out `'a` here.

Maybe the iterator and method should be named just `ones`? What do you think?